hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/db/Table.java

```diff
@@ -271,6 +271,15 @@ List<? extends KeyValue<KEY, VALUE>> getSequentialRangeKVs(KEY startKey,
           MetadataKeyFilters.MetadataKeyFilter... filters)
           throws IOException, IllegalArgumentException;
 
+  /**
+   * Delete all keys with a prefix.
+   * @param batch
+   * @param prefix
+   * @return
+   */
+  void deleteAllWithPrefix(BatchOperation batch, KEY prefix)
```

Comment:
deleteAllWithPrefix -> deleteWithBatch

Reply:
OK.

Reply:
deleteWithBatch seems to be an existing API, let's name it deleteBatchWithPrefix